dist/async-test.js

```diff
@@ -59,29 +59,13 @@
 	            var _this = this;
 	            if (!(this._pendingMicroTasks || this._pendingMacroTasks)) {
 	                // We do this because we would like to catch unhandled rejected promises.
-	                // To do this quickly when there are native promises, we must run using an unwrapped
-	                // promise implementation.
-	                var symbol = Zone.__symbol__;
-	                var NativePromise = window[symbol('Promise')];
-	                if (NativePromise) {
-	                    NativePromise.resolve(true)[symbol('then')](function () {
-	                        if (!_this._alreadyErrored) {
-	                            _this.runZone.run(_this._finishCallback);
+	                this.runZone.run(function () {
+	                    setTimeout(function () {
```

Comment:
I don't think so, because we don't want to trace our own setTimeout, but I also am not sure.